Update configure-deployment-resources.md (#2695)
astronomer/docs: astro/configure-deployment-resources.md

2. Click **Resource quotas**.
3. Configure the following values:

    - **Quota (CPU)**: The maximum amount of CPU for all currently running Pods on your Deployment. 
    - **Quota (Memory)**: The maximum amount of memory for all currently running Pods on your Deployment.
    - **Default Pod Size (CPU)**: The amount of CPUs that your tasks run with if no CPU usage is specified in their Pod configuration.

>     - **Default Pod Size (Memory)**: The amount of memory that your tasks run with if no memory usage is specified in their Pod configuration.

Reviewer: The field says "CPU Quota" instead of "Quota (CPU)". Bold styling in our docs mean "You could ctrl+f

Would an ellipses work better? Updated my suggestion, let me know what you think

>     - **Default Pod Size (Memory)**: The amount of memory that your tasks run with if no memory usage is specified in their Pod configuration.

Author: Nested bullets would be fine. I think a colon would be better than an ellipses or `>` character

:::warning

If the max pod size is equal to the deployment's quota, and a task requests the max pod size, only one task will run at a time, because that task is consuming the deployment's entire quota.

> :::

Author: > we can inclide it out of a note.

Do you mean like an info block? I think that'd be fine. I just think it ought to be highlighted.

> The CPU and memory quotas determine the **Max Pod Size**, which is the maximum amount of resources that a task can request for its pod.

Reviewer: ```suggestion
Your CPU and memory quotas determine how many tasks you can run at once on your Deployment. For example, if a task requests the full amount of your CPU and memory quotas, then your Deployment can run only that task until it completes.

The CPU and memory quotas also determine the **Max Pod Size**, which is the maximum amount of resources that a task can request for its pod.
```

> 

Reviewer: ```suggestion
:::caution

The largest possible **Max Pod Size** has fewer resources than the largest possible CPU and memory quotas. If your CPU and memory quotas exceed your **Max Pod Size**, the **Max Pod Size** becomes the maximum amount of resources you can request in a single task. 

For example, if you have a CPU quota of 150 and a memory quota of 200, your **Max Pod Size** will be 12 vCPU and 24GiB RAM. If you try to run a task that requests 20vCPU, the task won't run even though it's within your quotas.

:::

```

Author: cpu to memory is always 1:2, so this example would have to be "150 vCPU and a memory quota of 300GiB". My concern here is the difference in max pod size between Standard/Dedicated (and your point about "what if these maximums change?")

Reviewer: Updated my suggestion to afford the possibility that the values could change in the future. I think it's okay to use these values as examples even though the exact ratios might change over time. The main point is "Quotas can be greater than max pod size"

2. Click **Resource quotas**.
3. Configure the following values:

    - **Quota (CPU)**: The maximum amount of CPU for all currently running Pods on your Deployment. 
    - **Quota (Memory)**: The maximum amount of memory for all currently running Pods on your Deployment.
    - **Default Pod Size (CPU)**: The amount of CPUs that your tasks run with if no CPU usage is specified in their Pod configuration.
    - **Default Pod Size (Memory)**: The amount of memory that your tasks run with if no memory usage is specified in their Pod configuration.

Your CPU and memory quotas determine how many tasks can run at once on your Deployment. For example, if your Deployment has a CPU quota of 3vCPU and a memory quota of 6GiB, and a task requests this amount, then your Deployment can run only that task until it completes.

> The CPU and memory quotas also determine the **Max Pod Size**, which is the maximum amount of resources that a task can request.

Reviewer: ```suggestion
The CPU and memory quotas also determine the **Max Pod Size**, which is the maximum amount of resources that a task can request.

```

2. Click **Resource quotas**.
3. Configure the following values:

    - **CPU Quota**: The maximum combined CPU usage across all running Pods on your Deployment. 
    - **Memory Quota**: The maximum combined memory usage across all running Pods on your Deployment.
    - **Default Pod Size**:
        - **CPU**: The amount of CPUs that your tasks run with if no CPU usage is specified in their Pod configuration.
        - **Memory**: The amount of memory that your tasks run with if no memory usage is specified in their Pod configuration.

Your CPU and memory quotas determine how many tasks can run at once on your Deployment. For example, if your Deployment has a CPU quota of 3vCPU and a memory quota of 6GiB, and a task requests this amount, then your Deployment can run only that task until it completes.

The CPU and memory quotas also determine the **Max Pod Size**, which is the maximum amount of resources that a task can request.

:::caution

> The largest possible CPU and memory quotas can exceed the largest possible **Max Pod Size**. Because tasks can only run in a single Pod, your tasks can't request resources that exceed the **Max Pod Size**.

Reviewer: ```suggestion
The largest possible CPU and memory quotas can exceed the largest possible **Max Pod Size**. Because tasks run in a single Pod, your tasks can't request resources that exceed the **Max Pod Size**, even if your quota is larger.
```

> The largest possible CPU and memory quotas can exceed the largest possible **Max Pod Size**. Because tasks run in a single Pod, your tasks can't request resources that exceed the **Max Pod Size**, even if your quota is larger.

Reviewer: ```suggestion
For Deployments running on dedicated clusters, the largest possible CPU and memory quotas can exceed the largest possible **Max Pod Size**. Because tasks run in a single Pod, your tasks can't request resources that exceed the **Max Pod Size**, even if your quota is larger.
```
